.github/workflows/windows.yml

```diff
@@ -0,0 +1,93 @@
+name: mingw-w64
+
+on:
+  pull_request:
+  push:
+
+jobs:
+  mingw-w64:
+    name: Build and test
+    runs-on: windows-latest
+    steps:
+      - name: Checkout Git branch
+        uses: actions/checkout@v2
+
+      - name: Cache Msys2
+        id: cache-msys2
+        uses: actions/cache@v1.1.0
+        with:
+          path: msys64
+          key: ${{ runner.os }}-msys64
+
+      - name: Get Msys2
+        if: steps.cache-msys2.outputs.cache-hit != 'true'
+        run: |
+          Invoke-WebRequest -Outfile msys2.tar.xz -Uri "https://github.com/msys2/msys2-installer/releases/download/nightly-x86_64/msys2-base-x86_64-latest.tar.xz"
+          xz -d msys2.tar.xz
+          tar -xf msys2.tar
+          rm msys2.tar
+
+      - name: Add Msys2 to Path
+        run: |
+          echo "::add-path::${env:GITHUB_WORKSPACE}/msys64/mingw64/bin"
+          echo "::add-path::${env:GITHUB_WORKSPACE}/msys64/usr/bin"
+
+      - shell: cmd
+        if: steps.cache-msys2.outputs.cache-hit != 'true'
+        run: bash -lc 'exit'
+
+      - name: Update Pacman
+        run: pacman -Syy --needed --ask=20 --noconfirm
+
+      - name: Cache pacman
+        id: cache-pacman
+        uses: actions/cache@v1.1.0
+        with:
+          path: msys64/var/cache/pacman/pkg
+          key: ${{ runner.os }}-pacman-${{ hashFiles('msys64/var/lib/pacman/sync/mingw64.db') }}
+          restore-keys: |
+            ${{ runner.os }}-pacman-
+
+      - name: Update Msys2 Packages
+        run: pacman -Suu --needed --ask=20 --noconfirm
+
+      - name: Install packages
+        env:
+          pkg: orc pkg-config cmake gcc libpng SDL gettext
+        run: pacman -S --needed --ask=20 --noconfirm make $env:pkg.Split(' ').ForEach({"mingw-w64-x86_64-$_"})
+
+      - name: "[LV] Run CMake"
+        run: |
+          cmake -B build_lv -S libvisual -G"Unix Makefiles" -DCMAKE_BUILD_TYPE=Debug -DBUILD_SHARED_LIBS=OFF -DENABLE_DOCS=yes -DENABLE_NLS=OFF -DENABLE_TESTS=yes -DCMAKE_INSTALL_PREFIX="${env:GITHUB_WORKSPACE}/msys64/mingw64"
+
+      - name: '[LV] Run "make"'
+        run: |
+          make -C build_lv -j2 VERBOSE=1
+
+      - name: '[LV] Run "make install"'
+        run: |
+          make -C build_lv install
+          find ${env:GITHUB_WORKSPACE}/msys64/mingw64/bin/lv-tool ${env:GITHUB_WORKSPACE}/msys64/mingw64/include/libvisual-0.5 ${env:GITHUB_WORKSPACE}/msys64/mingw64/lib/libvisual-0.5* ${env:GITHUB_WORKSPACE}/msys64/mingw64/lib/pkgconfig/libvisual-0.5.pc | sort
```

Comment:
This whitelist seems to go against the idea of getting a listing of all files installed, including files that we are not expecting to be installed. The existing GitHub action uses a custom isolated `CMAKE_INSTALL_PREFIX`. That way we would not mix with msys64 and would not need a whitelist like that. Does it cause problems that I am not aware of?